qiskit_experiments/calibration_management/parameter_value.py

```diff
@@ -71,6 +72,19 @@ def __post_init__(self):
 
         self.date_time = self.date_time.astimezone()
 
+        if isinstance(self.value, complex):
+            warnings.warn(
+                "Support of complex parameters is now pending deprecation, following the"
+                "same transition in Qiskit Terra's Pulse module. With the exception of"
+                "the Half Angle Calibration experiment all experiments will continue to"
```

Comment:
You should not mention particular experiment name because user may have own custom experiment using complex parameter.